swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -298,11 +312,21 @@ public final class Store: ObservableObject {
     self.configurationTimer = timer
   }
 
-  private func endConfigurationPolling() {
```

Comment:
We now unconditionally try to fetch configuration from the tunnel, silencing the error where we're not initialized. This is much simpler to reason about and will auto recover our configuration if the system extension or VPN configuration ever gets into a weird state and then resolves itself.